exercises/practice/twelve-days/twelve_days_test.cpp

```diff
@@ -0,0 +1,211 @@
+#include "twelve_days.h"
+#ifdef EXERCISM_TEST_SUITE
+#include <catch2/catch.hpp>
+#else
+#include "test/catch.hpp"
+#endif
+
+TEST_CASE("first day a partridge in a pear tree",
+          "[c0b5a5e6-c89d-49b1-a6b2-9f523bff33f7]") {
+    std::string expected =
```

Comment:
Although it can't be constexpr in C++17, it can still be `const` (your choice where to put the keyword, I'm an east-const advocate, but we don't have a strict rule in this repo AFAIK):
```suggestion
    std::string const expected =
```